### PR TITLE
User Registration/Login Function Added

### PR DESCRIPTION
**Pull request by aadixiiita**
Using PHP PDO the Registration and Login function for student is added as of now ,

### CODE REVIEW

**Comment by aadixiiita** [comment]
Done

File: usersignup.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html>
+<style>
```
Comment (pradeepgangwar) [inline usersignup.php]: remove this.

File: index.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
```
Comment (pradeepgangwar) [inline index.php]: remove this too

**Comment by aadixiiita** [comment]
Done

**Comment by pradeepgangwar** [comment]
@aadixiiita Export your database as sql file and place that sql file here. So that others who use this have the same database and don't have to make a new one for their localhost.

**Comment by aadixiiita** [comment]
Done

**Pull request merged**